psql2mysql/__init__.py

```diff
@@ -368,8 +380,8 @@ def do_prechecks(config, source, target):
         long_values = db.scanTableForLongTexts(table)
         if long_values:
             print("Table '%s' contains TEXT values that are more than %s "
-                  "characters long. This is incompatible with MariaDB setup.",
-                  table.name, MAX_TEXT_LEN)
+                  "characters long. This is incompatible with MariaDB setup." %
```

Comment:
do you know why we mix `print()` with `LOG`?

Reply:
I think `print` was meant as the CLI output that is always shown, while log should have optional log levels and can (probably) be redirected to the log file...
That's why I think we did it this way in the first time...

Reply:
Ironic itself is a corner case :o)
Lets leave it as is.